client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/Allocation.tsx

```diff
@@ -62,6 +63,23 @@ export const Allocation = ({
     queryData().then(({ data }) => {
       if (!data) return;
 
+      const packsizes = [
+        ...new Set(data.draftLines.map(line => line.packSize)),
+      ];
+
+      // if there is only one packsize, we should auto-allocate in packs for that size
+      let allocateInPacksize: AllocateInOption | undefined = undefined;
+      if (packsizes.length === 1) {
+        const packSize = packsizes[0];
+        if (packSize) {
```

Comment:
These multiple `if` statements are a bit messy but Typescript didn't seem to trust that packsize might not be defined. Pretty sure if can be cleaned up but also this works?